Soil load

mickwelli opened the pull request:
### Proposed changes
Demonstrate loading of iSDA data with dc.load

### Checklist (replace `[ ]` with `[x]` to check off)
- [X] Remove any unused Python packages from `Load packages`
- [X] Remove any unused/empty code cells
- [X] Remove any guidance cells (e.g. `General advice`)
- [X] Ensure that all code cells follow the [PEP8 standard](https://www.python.org/dev/peps/pep-0008/) for code. The `jupyterlab_code_formatter` tool can be used to format code cells to a consistent style: select each code cell, then click `Edit` and then one of the `Apply X Formatter` options (`YAPF` or `Black` are recommended)
- [X] Include relevant tags in the first notebook cell and re-use tags if possible
- [X] Ensure appropriate colour schemes  have been used to maximise accessibility for vision impairment. Test your images or learn more with [Coblis](https://www.color-blindness.com/coblis-color-blindness-simulator/) or [TPGI](https://www.tpgi.com/color-contrast-checker/)
- [X] Clear all outputs, run notebook from start to finish, and save the notebook in the state where all cells have been sequentially evaluated

### Closes issues (optional)
- Closes Issue #396

fangfy reviewed (commented):
the loading looks good. Two minor issues:
* `fcc` is refered to in the description under "Define parameters" for loading indexed soil carbon data. 
* in the soil carbon plot (below exectued step 9), the std has a color ramp up to 1e6. Is there any outlier in the data loaded? should these values be exluded? or perhaps just force the plotting range?

lisarebelo reviewed (changes requested):
Thanks Michael, it looks good. I only request that the "Transformation!" headings to be more in line with the formatting of other section headings, e.g. "Apply a transformation factor" in same size font and colour without exclamation mark (unless there is a good reason for the current formatting?)

mickwelli commented:
Thanks @lisarebelo I've updated the formatting. Thanks @fangfy, yes that is weird with the std dev for soil total C - thanks for spotting. I've added `clip(0,1)` to that plot and explained why.
